src/REPLICA/fix_neb.cpp

```diff
@@ -849,6 +818,66 @@ void FixNEB::inter_replica_comm()
   }
 }
 
+/*
+Calculate ideal positions for parallel "ideal" or "equal"
+*/
+void FixNEB::calculate_ideal_positions()
+{
+  // Skip unless "ideal" or "equal"
+  if (not (EqualForceNEB or NEBLongRange)) return;
```

Comment:
Same as above, also "not" is not portable and not following the established convention in LAMMPS.

Reply:
Noted. Thank you!